app/src/androidTest/java/org/xwiki/android/sync/auth/AuthenticatorActivityTest.kt

```diff
@@ -69,29 +67,35 @@ class AuthenticatorActivityTest : LifecycleObserver {
         }
         onView(withId(R.id.btViewSignInFlipper)).perform(click())
         onView(withId(R.id.accountName))
-            .perform(typeText("TestUser"))      // Test user, for log in
+            .perform(typeText("aa700"))      // Test user, for log in
```

Comment:
it would be better to replace these strings to `private const val` at the root if this file

Reply:
Okay.